test/Java8andUp/src/org/openj9/test/vmArguments/VmArgumentTests.java

```diff
@@ -366,6 +414,14 @@ public void testJarArguments() {
 		assertTrue("Wrong order of properties", longPropPosn == fooBarPropPosn + 1);
 	}
 
+	private Path trimLeadingSlash(URL fileURL) {
```

Comment:
Use `Paths.get(url.toURI()).toFile()` instead of function.

Reply:
Actually I did:
Paths.get(jarUrl.toURI()).toFile().toPath()
because Files.copy() takes a Path not a File.
This pull request is getting pretty old, so I should retest.  Please stand by.